granulate_utils/linux/process.py

```diff
@@ -81,6 +81,13 @@ def read_process_execfn(process: psutil.Process) -> str:
 
 def _read_process_auxv(process: psutil.Process, auxv_id: int) -> int:
     auxv = read_proc_file(process, "auxv")
+    if not auxv:
+        # kernel thread / zombie process yield this result.
```

Comment:
Can't it be dead as well?

Reply:
If it's dead, there's no proc file.

Reply:
Yup, we'll get `NoSuchProcess` which is also good.

Callers of this function will need to handle `NoSuchProcess` & `ZombieProcess` the way the please.